tests/phpunit/elementor/test-elements.php

```diff
@@ -106,4 +108,19 @@ public function test_controlsDefaultData() {
 			}
 		}
 	}
+
+	private function removeCommasFromSelectorBrackets($selector)
+	{
+		$isContainCommaRegex = '/\([^(]+,[^(]+\)/';
+		preg_match( $isContainCommaRegex, $selector, $matches );
+
+		foreach($matches as $match) {
+			$matchWithoutCommas =str_replace(',', '', $match);
+			$selector = str_replace($match, $matchWithoutCommas, $selector);
+			return $this->removeCommasFromSelectorBrackets($selector);
+		}
```

Comment:
Why do you need that recursion? I think `preg_replace_callback` can do that simpler and more readable